silo/forms.py

```diff
@@ -29,6 +30,10 @@ class SiloForm(forms.ModelForm):
     def __init__(self, user=None, *args, **kwargs):
         super(SiloForm, self).__init__(*args, **kwargs)
         # Filter programs based on the program teams from Activity
+
+        if user != None:
```

Comment:
Comparison with `None` should be always done with `is` or `is not`.

Reply:
There's already a conditional with `user` in this form. Why are you creating another one?